code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -46,16 +84,17 @@
 	heldname = new_heldname
 	user.log_message("set \"[heldname]\" as a name in a cyborg reclassification board at [loc_name(user)]", LOG_GAME)
 
-/obj/item/borg/upgrade/rename/action(mob/living/silicon/robot/R, user = usr)
+/obj/item/borg/upgrade/rename/action(mob/living/silicon/robot/borg, user = usr)
 	. = ..()
-	if(.)
-		var/oldname = R.real_name
-		var/oldkeyname = key_name(R)
-		R.custom_name = heldname
-		R.updatename()
-		if(oldname == R.real_name)
-			R.notify_ai(AI_NOTIFICATION_CYBORG_RENAMED, oldname, R.real_name)
-		usr.log_message("used a cyborg reclassification board to rename [oldkeyname] to [key_name(R)]", LOG_GAME)
+	if(!.)
+		return .
```

Comment:
```suggestion
		return
```
I'm not sure if that level of clearness is one of those things san7890 would suggest... but if that's the case, please do ignore this, otherwise make sure to do this for every other `return .` in the file.